guides/v2.1/cloud/cdn/fastly-vcl-wordpress.md

```diff
@@ -11,62 +11,127 @@ functional_areas:
+
+    -  Add and save key-value pairs in the new dictionary. For this example, each **Key** is a URL path that you want to redirect to the WordPress backend, and the **Value** is 1. 
+       
+	   ![Add Edge Dictionary Items]
+	 
+    -  Click **Cancel** to return to the system configuration page.
+
+
+## Create a VCL snippet for the WordPress redirect {#vcl}
+
+The following custom VCL snippet code (JSON format) evaluates incoming requests to determine whether they require redirection to the WordPress backend you added to the Fastly service configuration.
 
 ```json
 {
-  "name": "wordpress",
+  "name": "wordpress_redirect",
   "dynamic": "0",
   "type": "recv",
   "priority": "5",
-  "content": "if ( req.url.path ~ \"^\\/?([^:\/\\s]+).*$\" ) { if ( table.lookup(wordpress_urls, re.group.1, \"NOTFOUND\") != \"NOTFOUND\" ) { set req.http.X-WP = \"1\"; } }"
+  "content": "if ( req.url.path ~ \"^\\/?([^\/?]+)") { if ( table.lookup(wordpress_urls, re.group.1, \"NOTFOUND\") != \"NOTFOUND\" ) { set req.http.X-WP = \"1\"; } }"
 }
-```
+````
+
+Review the code to determine if you need to change any values:
+
+  -  `name`: Name for the VCL snippet. For this example, we used the name `wordpress_redirect`.
+  
+  -  `dynamic`: Value 0 indicates that this is a [regular snippet](https://docs.fastly.com/guides/vcl-snippets/using-regular-vcl-snippets)  that you upload to the versioned VCL for the Fastly configuration.
+
+  -  `priority`: Determines when the VCL snippet runs. The priority is set to 5, which means that this snippet code runs before any of the default Magento VCL snippets (magentomodule) that have a priority of 50.
+
+  -  `type`: This VCL is a `recv` snippet type which adds the snippet code to the `vcl_recv`subroutine below the boilerplate VCL and above any objects.
+
+  -  `content`: When the VCL code in this example runs, it extracts the first part `mypath` of the path `/mypath/someotherpath`, and then compares that path to the values in the `wordpress_urls` dictionary. If a match is found, the request is redirected to the WordPress backend. See the [Fastly VCL reference](https://docs.fastly.com/vcl/reference/) for information about creating Fastly VCL code snippets.
+ 
+
+You can add the custom VCL snippet to your Fastly service configuration from the Admin UI (requires Fastly module 1.2.58 or later). If you do not have access to the Admin UI, you can save the JSON code example in a file and upload it using the Faslty API. See [Creating a VCL snippet using the Fastly API](https://docs.fastly.com/vcl/vcl-snippets/using-regular-vcl-snippets/#via-the-api). 
+
+
+#### To add the custom VCL snippet from the Admin UI
+
+1.  From the Magento Admin UI, navigate to **Stores** > **Configuration** > **Advanced** > **System** > **Fastly Configuration**.
+
+1.  Expand the **Custom VCL snippet** section.
+
+1.  Click **Create Custom Snippet**.
+
+1.  Complete the snippet form as shown in the following figure:
+
+    ![Create VCL Snippet]
```

Comment:
The snippet here can't be copied and pasted as it's an image. Can this be presented as text that can be copied somehow?